mex/common/settings.py

```diff
@@ -167,7 +167,7 @@ def get(cls) -> Self:
         validation_alias="MEX_PRIMARY_SOURCES_PATH",
     )
     ldap_url: SecretStr = Field(
-        SecretStr("ldap://user:pw@ldap:636"),
+        SecretStr("ldap://user:pw@host:port"),
```

Comment:
`ldap` is already a placeholder, and the port has to be an integer i suspect. i'd say we can stick with the protocol's default port here

```suggestion
        SecretStr("ldap://user:pw@ldap:639"),
```